crates/router/src/core/payouts/helpers.rs

```diff
@@ -47,10 +47,10 @@ pub async fn make_payout_method_data<'a>(
             )?;
             Ok(pm)
         }
-        (Some(payout_method), None) => {
+        (Some(payout_method), payout_token) => {
```

Comment:
it should be None `store_payout_method_data_in_locker ` will give you the token and we need to update that in `payout_create`

Reply:
For the cases where payout_method_data is present in the request and payout_token is also present in DB, it will return None.

This case was failing during Update flow.

We need to either disallow changing the payout_type and method_data once a payout is created, or else we need to handle it using the changes above